### PR TITLE
kubeadm: updated preflight types to avoid stutter

### PR DESCRIPTION
**Pull request by dmmcquay**
Small change to kubeadm preflight pkg to remove stutter from preflight types PreFlightError and PreFlightCheck (now names Error and Checker). 

**Release note**:

`NONE`

### CODE REVIEW

**Review by luxas — APPROVED** [review]
LGTM when the build passes.
@jbeda PTAL

**Comment by luxas** [comment]
@k8s-bot ok to test

**Comment by luxas** [comment]
@pires If you also LGTM, I'm gonna apply the label, this is a straightforward PR

**Review by pires — APPROVED** [review]
@luxas lgtm

**Comment by dmmcquay** [comment]
@k8s-bot verify test this

**Comment by pires** [comment]
Why did you merge master into the branch? I think you'll need to redo the PR.

**Comment by dmmcquay** [comment]
@pires because it had a merge conflict with upstream master.

**Comment by dmmcquay** [comment]
@k8s-bot gci gce e2e test this

**Comment by dmmcquay** [comment]
@k8s-bot gce etcd3 e2e test this

**Comment by dmmcquay** [comment]
@pires @luxas all checks have passed, just waiting for lgtm after solving the merge conflict.

**Pull request merged**
